packages/itwin/ec3-widget/src/components/EC3/EC3TokenCache.ts

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+export interface EC3TokenCache {
```

Comment:
This looks more like a cache entry than a cache.

Reply:
renamed to EC3Token